docker/README.md

```diff
@@ -29,7 +29,7 @@ docker run --net=host --name='myriad-resourcemanager' -t \
 
 If you already had a working Vagrant instance, you will need to run `vagrant reload` in order to allow zookeeper and hdfs port-forwarding.
 
-Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](https://github.com/mesos/myriad/blob/phase1/myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
+Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](../myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
 
```

Comment:
Also, in general I place myriad-config-default.yml in $HADOOP_HOME/etc/hadoop/, this puts it in the classpath before the one packaged in `srv/main/resources/`, and allows for configuration without recompilation.  I think this should be the preferred method and the documentation should reflect that.